index.d.ts

```diff
@@ -177,6 +194,7 @@ declare module "react-native-twilio-video-webrtc" {
     publishLocalVideo: () => void;
     unpublishLocalVideo: () => void;
     sendString: (message: string) => void;
+    sendEncodedString: (message: string) => void;
```

Comment:
Took out sendEncodedString since I realized the ticket only specifies receiving data, but I will keep this in mind when we tackle it.

Reply:
You were so close though! 😞 But I get it. We've got lots of other stuff to do.

It does mean that we probably can't submit this upstream until the sending part is implemented, since I doubt they'd want to merge in a PR that only supports one side of the communication.